admin-manual/installation-setup/installation/installation.rst

```diff
@@ -0,0 +1,202 @@
+.. _installation:
+
+========================
+Installing Archivematica
+========================
+
+*On this page*
+
+* :ref:`Overview <overview>`
+* :ref:`Technical requirements <tech-requirements>`
+* :ref:`Instructions for new installations <instructions>`
+* :ref:`Advanced installation options <advanced>`
+
+.. _overview:
+
+Overview
+========
+
+Archivematica is not a single application - dozens of different components and
+tools are required for a fully working installation. As a result, there are many
+possible deployment configurations.
+
+These instructions are designed to get you up and running as quickly as
+possible, even if you are not familiar the various tools and applications that
+are bundled into Archivematica. Experience with the Linux command line is
+helpful, and to support a running production system it should be considered a
+requirement.
+
+If you need assistance or clarification regarding the installation instructions,
+the `Archivematica google group`_ is a good place ask questions.
+
+.. note:: For testing purposes, you may find it easier to install on a virtual
+   machine using Vagrant. See the :ref:`Quick Start Guide <quick-start-install>`
+
+.. _tech-requirements:
+
+Technical requirements
+======================
+
+Operating system
+----------------
+
+Archivematica |release| installation instructions are provided here for the
+following operating systems:
+
+* Ubuntu 14.04.5 64-bit Server Edition
+* Ubuntu 16.04.2 64-bit Server Edition (beta)
+* CentOS 7.3.1611 64-bit
+
+Archivematica |version| is the first release to be tested on Ubuntu 16.04. Support
+for this OS is still considered beta; installation has been tested but production
+deployments are limited.
+
+Other Linux distributions should work, but will require customization of these
+installation instructions.
+
+Support for Mac OS X is possibly in theory, but is not being tested, and would
+require more significant deviation from these instructions.
+
+Archivematica is unlikely to ever run directly in a Windows environment.
+Consider the use of a virtualization platform to run Linux VMs.
+
+Dependencies
+------------
+
+Archivematica has a long list of software it depends on. All of these
+dependencies are installed when following the instructions below.
+
+In these instructions everything is installed into one machine. It is possible
+to install some of the components on separate machines, to improve performance,
+such as:
+
+* MySQL
+* gearman
+* Elasticsearch (optional as of Archivematica 1.7, see below)
+
+Archivematica has a long list of software it depends on. All of these
+dependencies are installed when following the instructions below.
+
+In these instructions everything is installed into one machine. It is possible
```

Comment:
Perhaps this sentence is now not accurate in this page?

Reply:
Suggestion for improving the accuracy? Could also just remove it.